src/PresentationalComponents/Tailorings/Tailorings.js

```diff
@@ -186,11 +175,14 @@ const Tailorings = ({
                   {...{
                     ...(tab.isSecurityGuide
                       ? {
-                          securityGuide: tab,
+                          securityGuideId: tab.securityGuideId,
```

Comment:
I might be wrong, but it feels it needs to be
```suggestion
                          securityGuideId: tab.security_guide_id,
```

Reply:
It is `securityGuideId`, this is coming from here: https://github.com/RedHatInsights/compliance-frontend/pull/2270/files#diff-34f9c79006751759870ff27bee13795d1275721b209782132c7af48d754a99a7R91